tests/test_actor.py

```diff
@@ -237,3 +238,45 @@ def test_actor_processes_all_messages_before_stop_on_self_stops_it(actor_ref, ev
 
     events.on_stop_was_called.wait(5)
     assert len(ActorRegistry.get_all()) == 0
+
+
+def test_delayed_message_is_actually_delayed(runtime, actor_ref):
+    event = runtime.event_class()
+    lst = []
+
+    actor_ref.tell(
+        {
+            "command": "callback",
+            "callback": lambda: lst.append(2) or event.set(),
+        },
+        delay=1,
+    )
+    actor_ref.tell(
+        {"command": "callback", "callback": lambda: lst.append(1)}, delay=0.5
+    )
+    event_set = event.wait()
```

Comment:
Should this have some largish timeout just in case, so the test exists even if this breaks at some point?